src/rust/ensogl/lib/components/src/card.rs

```diff
@@ -0,0 +1,71 @@
+//! This module provides the [`Card`] component.
+
+use crate::prelude::*;
+
+use crate::shadow;
+
+use ensogl_core::display;
+use ensogl_core::display::shape::*;
+use ensogl_core::display::object::{ObjectOps, Instance};
+use ensogl_theme as theme;
+
+
+
+// ============
+// === Card ===
+// ============
+
+/// The estimated margin that we need on all sides to include the shadow.
+const SHADOW_PX:f32 = 10.0;
+
+ensogl_core::define_shape_system! {
+    (style:Style,corner_radius:f32) {
+        let sprite_width  : Var<Pixels> = "input_size.x".into();
+        let sprite_height : Var<Pixels> = "input_size.y".into();
+
+        let width  = sprite_width - SHADOW_PX.px() * 2.0;
+        let height = sprite_height - SHADOW_PX.px() * 2.0;
+        let color  = style.get_color(theme::graph_editor::node::background);
```

Comment:
In this case, we are defining a card shape and this is the color of the card. `bg_color` would be confusing, in my opinion. Maybe we should call it `card_color`. But that seems a bit redundant. Do you have a better suggestion?

Reply:
By the way, such "fix it everywhere" comments take a very long time for me, especially since I am often unaware why a name is not clear to the reader. It would safe me a lot of time if you left just a short comment whenever a name is unclear to you. Suggested improvements or other thoughts would also be helpful, of course.

Reply:
I created an issue for it (#1706).

Reply:
The names "color" are not semantically descriptive. It is like naming a variable "var". Sure it is a color, but what this color is used for? `bg_color` is million times better, nad yes, I'd use `bg_color`. Please remember that this should be fixed as the second PR here. It means this should probably be applied by @MichaelMauderer now then